GPU/Common/TextureReplacer.cpp

```diff
@@ -271,37 +271,37 @@ bool TextureReplacer::LoadIniValues(IniFile &ini, VFSBackend *dir, bool isOverri
 		// Format: hashname = filename.png
 		bool checkFilenames = g_Config.bSaveNewTextures && !g_Config.bIgnoreTextureFilenames && !vfsIsZip_;
 
-		for (const auto &item : hashes) {
+		for (const auto &[k, v] : hashes) {
 			ReplacementCacheKey key(0, 0);
 			// sscanf might fail to pluck the level if omitted from the line, but that's ok, we default level to 0.
 			// sscanf doesn't write to non-matched outputs.
 			int level = 0;
-			if (sscanf(item.first.c_str(), "%16llx%8x_%d", &key.cachekey, &key.hash, &level) >= 1) {
+			if (sscanf(k.c_str(), "%16llx%8x_%d", &key.cachekey, &key.hash, &level) >= 1) {
 				// We allow empty filenames, to mark textures that we don't want to keep saving.
-				filenameMap[key][level] = item.second;
+				filenameMap[key][level] = v;
 				if (checkFilenames) {
 					// TODO: We should check for the union of these on all platforms, really.
 #if PPSSPP_PLATFORM(WINDOWS)
 					bool bad = item.second.find_first_of("\\ABCDEFGHIJKLMNOPQRSTUVWXYZ:<>|?*") != std::string::npos;
```

Comment:
D:\a\ppsspp\ppsspp\GPU\Common\TextureReplacer.cpp(285,17): error C2065: 'item': undeclared identifier

Reply:
@Nemoumbra I built PPSSPP on Debian, I didn't check it on Windows. I'll fix it now.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18559/commits/63337c5a53a40af95b114c094550b9eda8dab0cd